dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -349,6 +351,170 @@ public void testEmbargoedBitstream() throws Exception {
     }
 
+    @Test
+    public void embargoedBitstreamForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity)
+                .withName("Collection 1")
+                .build();
+
+        // a public item with an embargoed bitstream
+        String bitstreamContent = "Embargoed!";
+
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+
+            Item publicItem1 = ItemBuilder.createItem(context, col1)
+                                          .withTitle("Public item 1")
+                                          .withIssueDate("2017-10-17")
+                                          .withAuthor("Smith, Donald")
+                                          .build();
+
+            Bitstream bitstream = BitstreamBuilder
+                .createBitstream(context, publicItem1, is)
+                .withName("Test Embargoed Bitstream")
+                .withDescription("This bitstream is embargoed")
+                .withMimeType("text/plain")
+                .withEmbargoPeriod("3 months")
+                .build();
+
+            context.restoreAuthSystemState();
+
+            String authToken = getAuthToken(eperson.getEmail(), password);
+            getClient(authToken).perform(get("/api/core/bitstreams/" + bitstream.getID() + "/content"))
+                       .andExpect(status().isForbidden());
+
+            // An forbidden request should not log statistics
+            checkNumberOfStatsRecords(bitstream, 0);
+       }
+    }
+
+    @Test
+    public void expiredEmbargoedBitstreamTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity)
+                .withName("Collection 1")
+                .build();
+
+        String bitstreamContent = "Embargoed!";
+
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+
+            Item publicItem1 = ItemBuilder.createItem(context, col1)
+                    .withTitle("Public item 1")
+                    .withIssueDate("2015-10-17")
+                    .withAuthor("Smith, Donald")
+                    .build();
+
+            Bitstream bitstream = BitstreamBuilder
+                .createBitstream(context, publicItem1, is)
+                .withName("Test Embargoed Bitstream")
+                .withDescription("This bitstream is embargoed")
+                .withMimeType("text/plain")
+                .withEmbargoPeriod("-3 months")
+                .build();
+
+            context.restoreAuthSystemState();
```

Comment:
Again, here, I think the `try` can be closed *after* the BitstreamBuilder. Everything else can move outside of the `try` clause. Look at how you did this in `embargoedBitstreamAccessGrantByAdminsTest()`